src/main/java/teammates/common/util/NationalityHelper.java

```diff
@@ -0,0 +1,284 @@
+        addToNationalitiesList("Romanian");
+        addToNationalitiesList("Russian");
+        addToNationalitiesList("Rwandan");
+        addToNationalitiesList("Saint Lucian");
+        addToNationalitiesList("Salvadoran");
+        addToNationalitiesList("Samoan");
+        addToNationalitiesList("San Marinese");
+        addToNationalitiesList("Sao Tomean");
+        addToNationalitiesList("Saudi");
+        addToNationalitiesList("Scottish");
+        addToNationalitiesList("Senegalese");
+        addToNationalitiesList("Serbian");
+        addToNationalitiesList("Seychellois");
+        addToNationalitiesList("Sierra Leonean");
+        addToNationalitiesList("Singaporean");
+        addToNationalitiesList("Slovakian");
+        addToNationalitiesList("Solomon Islander");
+        addToNationalitiesList("Somali");
+        addToNationalitiesList("South African");
+        addToNationalitiesList("South Korean");
+        addToNationalitiesList("Spanish");
+        addToNationalitiesList("Sri Lankan");
+        addToNationalitiesList("Sudanese");
+        addToNationalitiesList("Surinamer");
+        addToNationalitiesList("Swazi");
+        addToNationalitiesList("Swedish");
+        addToNationalitiesList("Swiss");
+        addToNationalitiesList("Syrian");
+        addToNationalitiesList("Taiwanese");
+        addToNationalitiesList("Tajik");
+        addToNationalitiesList("Tanzanian");
+        addToNationalitiesList("Thai");
+        addToNationalitiesList("Togolese");
+        addToNationalitiesList("Tongan");
+        addToNationalitiesList("Tobagonian");
+        addToNationalitiesList("Tunisian");
+        addToNationalitiesList("Turkish");
+        addToNationalitiesList("Tuvaluan");
+        addToNationalitiesList("Ugandan");
+        addToNationalitiesList("Ukrainian");
+        addToNationalitiesList("Uruguayan");
+        addToNationalitiesList("Uzbekistani");
+        addToNationalitiesList("Venezuelan");
+        addToNationalitiesList("Vietnamese");
+        addToNationalitiesList("Welsh");
+        addToNationalitiesList("Yemenite");
+        addToNationalitiesList("Zambian");
+        addToNationalitiesList("Zimbabwean");
+        addToNationalitiesList("Other");
+    }
+    
+    /**
+     * This prevents instantiation. This is necessary because this is a utility class.
+     */
+    private NationalityHelper() {
+    }
+
+    /**
+     * Adds the nationalities to the NATIONALITIES list.
+     */
+    public static void addToNationalitiesList(String...nationalities) {
+        for (String nationality : nationalities) {
+            NATIONALITIES.add(nationality);
+        }
+    }
+
+    /**
+     * Returns with the nationalities list.
+     */
+    public static List<String> getNationalities() {
+        return NATIONALITIES;
+    }
+    
+    /**
+     * Returns the nationalities as HTML code.
+     */
+    public static List<ElementTag> getNationalitiesAsElementTags(String existingNationality) {
```

Comment:
Element tag is a UI-specific function; utility classes such as this one (`NationalityHelper`) should not have business with it.
If you need the nationalities list as list of element tags, do it in `PageData`. This way we achieve separation of concern.